server/typings/models/video/video-abuse.ts

```diff
@@ -16,12 +16,12 @@ export type MVideoAbuseId = Pick<VideoAbuseModel, 'id'>
 export type MVideoAbuseVideo =
   MVideoAbuse &
   Pick<VideoAbuseModel, 'toActivityPubObject'> &
-  Use<'Video', MVideo>
+  Use<'Video', MVideoAccountLightBlacklistAllFiles>
```

Comment:
Did you check references of this definition have these extra Video fields?

Reply:
Indeed, they don't for `MVideoAbuseVideo`.